src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,23 @@
 // your components will all go in this `component` directory.
 // feel free to change this component.js into TodoList.js
+
+
+// - `<TodoList />` receives your Todos array and iterates over the list generating a new `<Todo />` for each element in the array.
+
+
+import React from 'react'
+  
+const TodoList = [
```

Comment:
TodoList should be a functional component. This array is correct, but it would help to put it on the App component's state. The array can then be passed down to the TodoList component to be rendered.